gui/system/migrations/0009_auto_20170926_1530.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-09-26 15:30
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('system', '0008_auto_20170906_2335'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='settings',
+            name='stg_sysloglevel',
+        ),
+        migrations.AddField(
+            model_name='advanced',
+            name='adv_sysloglevel',
+            field=models.CharField(choices=[('f_emerg', 'Emergency'), ('f_alert', 'Alert'), ('f_crit', 'Critical'), ('f_err', 'Error'), ('f_warning', 'Warning'), ('f_notice', 'Notice'), ('f_info', 'Info'), ('f_debug', 'Debug'), ('f_is_debug', 'Is_Debug')], default='f_info', help_text='Specifies which messages will be logged by server. INFO and VERBOSE log transactions that server performs on behalf of the client. f_is_debug specify higher levels of debugging output. The default is f_info.', max_length=120, verbose_name='Syslog level'),
+        ),
+    ]
```

Comment:
Please remove and do all of them in a single migration.

Also you forgot again to migrate values from old model. Like this you're losing the current values.